suite-native/module-accounts-import/src/components/SelectableNetworkItem.tsx

```diff
@@ -60,7 +60,7 @@ export const SelectableNetworkItem = ({
                         <Text variant="body">{cryptoCurrencyName}</Text>
                         <Box flexDirection="row" alignItems="center">
                             <Text variant="hint" color="gray600">
-                                {CurrencySymbolFormatter.format(cryptoCurrencySymbol)}
+                                {NetworkSymbolFormatter.format(cryptoCurrencySymbol)}
```

Comment:
Prefer component version of formatters.

Reply:
https://github.com/trezor/trezor-suite/pull/7401/commits/1cc34593cd42d928be9c517ad6ad4da0f00dc9a3